controllers/ingressnodefirewall_controller_test.go

```diff
@@ -98,10 +97,10 @@ var _ = Describe("IngressNodeFirewall controller", func() {
 		Expect(k8sClient.DeleteAllOf(context.Background(), &infv1alpha1.IngressNodeFirewall{})).Should(Succeed())
 		Expect(k8sClient.DeleteAllOf(context.Background(), &v1.Node{})).Should(Succeed())
 		// Ignore any not found errors.
-		k8sClient.DeleteAllOf(
+		Expect(k8sClient.DeleteAllOf(
```

Comment:
That's was on purpose, though, see comment:
~~~
// Ignore any not found errors.
~~~

One of the tests delets all IngressNodeFirewallNodeState objects and thus DeleteAllOf will throw and error as it doesn't find any object. Did you rerun `make test` and it still passed? In that case, I'm o.k. and my assumption was wrong. But the Expect here yesterday made the test fail for me, so I removed it.

So perhaps _ := k8sClient.DeleteAllOf will make the linter happy while still simply ignoring the result here?